components/nimbus/src/enrollment.rs

```diff
@@ -669,9 +683,10 @@ impl<'a> EnrollmentsEvolver<'a> {
         // We need to mop up all the enrollments that aren't rollouts (not just belonging to experiments that aren't rollouts)
         // because some of them don't belong to any experiments recipes, and evolve_enrollment_recipes will handle the error
         // states for us.
-        let experiments_only = |e: &Experiment| !e.is_rollout();
-        let prev_experiments = filter_experiments(prev_experiments, experiments_only);
-        let next_experiments = filter_experiments(next_experiments, experiments_only);
+        let prev_experiments =
+            filter_experiments(prev_experiments, SharedExperimentMetadata::is_not_rollout);
+        let next_experiments =
+            filter_experiments(next_experiments, SharedExperimentMetadata::is_not_rollout);
```

Comment:
Oh, ok. I think I'd prefer seeing a closure which we use twice declared in this method.

Reply:
Using a single closure twice proved difficult because of the typing — it's not a fan of dynamic typing a lambda that supports both `&E` and `&Experiment`. Or at least I've not found a way to get it to support that.

I can totally replace it with a `|exp| !exp.is_rollout()` though